media-video/mpv/mpv-0.11.0-r1.ebuild

```diff
@@ -0,0 +1,278 @@
+		--docdir="${EPREFIX}"/usr/share/doc/${PF}
+
+		$(usex cli '' '--disable-cplayer')
+		$(use_enable libmpv libmpv-shared)
+
+		--disable-libmpv-static
+		--disable-static-build
+		--disable-build-date	# Create reproducible build
+		--disable-optimize		# Do not add '-O2' to CFLAGS
+		--disable-debug-build	# Do not add '-g' to CFLAGS
+
+		$(use_enable doc pdf-build)
+		$(use_enable vf-dlopen vf-dlopen-filters)
+		$(use_enable cli zsh-comp)
+		$(use_enable test)
+
+		$(use_enable iconv)
+		$(use_enable samba libsmbclient)
+		$(use_enable lua)
+		$(usex luajit '--lua=luajit' '')
+		$(use_enable libass)
+		$(use_enable libass libass-osd)
+		$(use_enable encode encoding)
+		$(use_enable bluray libbluray)
+		$(use_enable dvd dvdread)
+		$(use_enable dvd dvdnav)
+		$(use_enable cdda)
+		$(use_enable enca)
+		$(use_enable libguess)
+		--disable-uchardet		# Disable uchardet until #569114 is fixed
+		$(use_enable rubberband)
+		$(use_enable lcms lcms2)
+		--disable-vapoursynth	# Only available in overlays
+		--disable-vapoursynth-lazy
+		$(use_enable archive libarchive)
+
+		--enable-libavfilter
+		--enable-libavdevice
+
+		# Audio outputs
+		$(use_enable sdl sdl2)	# Listed under audio, but also includes video
+		--disable-sdl1
+		$(use_enable oss oss-audio)
+		--disable-rsound		# Only available in overlays
+		$(use_enable pulseaudio pulse)
+		$(use_enable jack)
+		$(use_enable openal)
+		$(use_enable alsa)
+		--disable-coreaudio
+		--disable-dsound
+		--disable-wasapi
+
+		# Video outputs
+		--disable-cocoa
+		$(use_enable wayland)
+		$(use_enable X x11)
+		$(use_enable xscreensaver xss)
+		$(use_enable X xext)
+		$(use_enable xv)
+		$(use_enable xinerama)
+		$(use_enable X xrandr)
+		$(use_enable opengl gl-x11)
+		$(usex egl "$(use_enable X egl-x11)" '--disable-egl-x11')
+		$(use_enable wayland gl-wayland)
+		$(use_enable vdpau)
+		$(usex vdpau "$(use_enable opengl vdpau-gl-x11)" '--disable-vdpau-gl-x11')
+		$(use_enable vaapi)
+		$(use_enable vaapi vaapi-vpp)
+		$(usex vaapi "$(use_enable opengl vaapi-glx)" '--disable-vaapi-glx')
+		$(use_enable libcaca caca)
+		$(use_enable drm)
+		$(use_enable jpeg)
+		$(use_enable raspberry-pi rpi)
+
+		# HWaccels
+		$(use_enable vaapi vaapi-hwaccel)
+		# Automagic VDPAU HW acceleration. See Gentoo bug 558870.
+
+		# TV features
+		$(use_enable v4l tv)
+		$(use_enable v4l tv-v4l2)
+		$(use_enable v4l libv4l2)
+		$(use_enable v4l audio-input)
+		$(use_enable pvr)
+		$(use_enable dvb dvbin)
+	)
+	waf-utils_src_configure "${mywafargs[@]}"
+}
+
+src_install() {
+	waf-utils_src_install
+
+	if use cli && use luajit; then
+		pax-mark -m "${ED}usr/bin/${PN}"
+	fi
+}
+
+pkg_preinst() {
+	gnome2_icon_savelist
+}
+
+pkg_postinst() {
+	fdo-mime_desktop_database_update
+	gnome2_icon_cache_update
+}
+
+pkg_postrm() {
+	fdo-mime_desktop_database_update
+	gnome2_icon_cache_update
+}
+
+src_test() {
+	cd "${S}"/build/test || die
+	for test in *; do
+		if [ -x "${test}" ]; then
+			$(${test}) || die "Test suite failed"
```

Comment:
I'm pretty sure this won't work the way you expect it to. In particular, if the file had `foo1; foo2; foo3` command, it would evaluate to:

    foo1; foo2; foo3 || die